packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1040,9 +1040,10 @@ export class ArcRotateCamera extends TargetCamera {
         }
 
         this.inputs.checkInputs();
-
         let hasUserInteractions = false;
 
+        // Scale speed by framerate to ensure higher framerate devices don't clamp movement to 0 too early
+        const speed = this._computeLocalCameraSpeed();

```

Comment:
Are you sure of the maths there ? I remember this being crazy involved in term of code from a long time ago when it was attempted.

Reply:
I do not think this is back compatible as well and it might impact the users setup ?